WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift

```diff
@@ -123,13 +119,6 @@ final class OrdersRootViewController: UIViewController {
         ServiceLocator.pushNotesManager.resetBadgeCount(type: .storeOrder)
     }
 
-    override var shouldShowOfflineBanner: Bool {
-        if featureFlagService.isFeatureFlagEnabled(.splitViewInOrdersTab) {
-            return false
-        }
-        return true
-    }
```

Comment:
Just for documentation: If we restore this offline banner along with the one in Order creation, when loading the split view we'll see a banner duplication:
![simulator_screenshot_116E73C6-9A29-42D6-AAFC-05424CBF3B3F](https://github.com/woocommerce/woocommerce-ios/assets/3812076/5a83be0c-8d71-4abc-8a67-d9cec01d4650)

Definitely not the best design, but I'm wondering if would be better to restore them both (specially for conveying this info the a merchant that is using an iPhone) than having none? It feels we're making the experience worst if we remove them rather than keeping them with not-so-good design. What do you think?